package.json

```diff
@@ -14,18 +14,27 @@
     },
     "scripts": {
         "lint-md": "remark content",
+        "package": "node scripts/package/package.js",
         "spell-md": "mdspell -a -n --en-us 'content/**/*.md'"
     },
     "dependencies": {
+        "gray-matter": "4.0.2",
+        "jsdom": "^12.2.0",
+        "js-yaml": "3.13.1",
+        "marked": "0.6.2",
```

Comment:
Definitely, we need to do this, and before you have a Markdown meltdown. I have filed https://github.com/mdn/sprints/issues/1505 for this, but feel free to edit it as you see fit. I have made it an epic because it looks like it might be bigger than a single user story. But maybe not.

I hope, again, that getting experience of migrating content will help us decide which features we need.